src/Libraries/RevitNodes/Schedules/ScheduleFilter.cs

```diff
@@ -0,0 +1,161 @@
+        public static ScheduleFilter ByFieldTypeAndValue(ScheduleField field, string filterType, object value)
+        {
+            var ft = (Autodesk.Revit.DB.ScheduleFilterType)System.Enum.Parse(typeof(Autodesk.Revit.DB.ScheduleFilterType), filterType);
+            Autodesk.Revit.DB.ScheduleFilter filter;
+
+            if (value.GetType() == typeof(int))
+            {
+                filter = new Autodesk.Revit.DB.ScheduleFilter(field.InternalScheduleField.FieldId, ft, (int)value);
+            }
+            else if (value.GetType() == typeof(double))
+            {
+                filter = new Autodesk.Revit.DB.ScheduleFilter(field.InternalScheduleField.FieldId, ft, (double)value);
+            }
+            else if (value.GetType() == typeof(string))
+            {
+                filter = new Autodesk.Revit.DB.ScheduleFilter(field.InternalScheduleField.FieldId, ft, (string)value);
+            }
+            else
+            {
+                filter = new Autodesk.Revit.DB.ScheduleFilter(field.InternalScheduleField.FieldId, ft, ((Revit.Elements.Element)value).InternalElement.Id);
+            }
+
+            return new ScheduleFilter(filter);
+        }
+
+        #endregion
+
+        /// <summary>
+        /// Filter Type
+        /// </summary>
+        public string FilterType
+        {
+            get
+            {
+                return this.InternalScheduleFilter.FilterType.ToString();
+            }
+        }
+
+        /// <summary>
+        /// Related Schedule Field Id
+        /// </summary>
+        public int FiledId
+        {
+            get
+            {
+                return this.InternalScheduleFilter.FieldId.IntegerValue;
+            }
+        }
+
+        /// <summary>
+        /// Value assigned to Schedule Filter
+        /// </summary>
+        public object Value
+        {
+            get
+            {
+                if (this.InternalScheduleFilter.IsStringValue)
+                {
+                    return this.InternalScheduleFilter.GetStringValue();
+                }
+                else if (this.InternalScheduleFilter.IsDoubleValue)
+                {
+                    return this.InternalScheduleFilter.GetDoubleValue();
+                }
+                else if (this.InternalScheduleFilter.IsIntegerValue)
+                {
+                    return this.InternalScheduleFilter.GetIntegerValue();
+                }
+                else if (this.InternalScheduleFilter.IsElementIdValue)
+                {
+                    return this.InternalScheduleFilter.GetElementIdValue().IntegerValue;
+                }
+                else
+                {
+                    return null;
+                }
+            }
+        }
+
+        public override string ToString()
+        {
+            return string.Format("ScheduleFilter(FilterType: {0}, FieldId: {1}, Value: {2})", this.FilterType, this.FiledId, this.Value);
```

Comment:
I don't think so since they are all required for the filter to be properly defined. The `value` can be an empty string for example but not null.